src/stage0.txt

```diff
@@ -12,15 +12,15 @@
 # source tarball for a stable release you'll likely see `1.x.0` for rustc and
 # `0.x.0` for Cargo where they were released on `date`.
```

Comment:
This comment seems outdated? At least on the current stable branch and also in this PR I see `1.x.0` and `0.(x+1).0`, unlike what the comment says.

Reply:
Hm yeah probably, I just go off of the forge docs. Mind filing a PR against master fixing this? (Or just an issue is fine too.)